active-record/src/main/java/com/iluwatar/activeobject/Rowverride.java

```diff
@@ -0,0 +1,12 @@
+package com.iluwatar.activeobject;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.SOURCE)
+public @interface Rowverride {
+
+}
```

Comment:
What is this for?

Reply:
Good question, its an interface used to override static methods.